boards/riscv/beagle_v_starlight/Kconfig.defconfig

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD
+	default "beagle_v_starlight"
```

Comment:
as per @pdp7's recommendation, I would change to `beaglev_starlight_jh7100` and rename the parent directory accordingly.

Reply:
Fixed.